index.js

```diff
@@ -50,7 +50,7 @@
           // thinking in terms of Dom node nesting, running counter
           // to React's "you shouldn't care about the DOM" philosophy.
           while(source.parentNode) {
-            found = (source === localNode || source.classList.contains(IGNORE_CLASS));
+            found = (source === localNode || source.classList && source.classList.contains(IGNORE_CLASS));
```

Comment:
this moves the bug from "crashes on safari" to "does the wrong thing on safari". I'd rather we try to really fix this rather than merely making it not error out. If Safari supports classList but just not for the SVG element, we need to find out for which versions that is the case, and whether that happens when the page doctype indicates HTML5 content through `<!doctype html>` - as a true HTML5 element, if SVG is lacking .classList support then that is a bug in Safari that also needs to be filed on the Safari issue tracker, which I believe is straight up webkit.